src/irmin/irmin.mli

```diff
@@ -1740,16 +1740,17 @@ module Private : sig
 
           {b Note:} Both [min] and [max] are subsets of [c].*)
 
-      val iter_on_closure :
+      val iter :
         [> `Read ] t ->
         min:node list ->
         max:node list ->
-        f_nodes:(node -> unit Lwt.t) ->
-        f_edges:(node -> node list -> unit Lwt.t) ->
+        ?node:(node -> unit Lwt.t) ->
+        ?edge:(node -> node list -> unit Lwt.t) ->
```

Comment:
why `node -> node list -> unit` and not `node -> node -> unit` ?

Reply:
I changed to `?edge:(node -> pred:node -> unit Lwt.t)` as the edges are directed. 
`pred` is clear enough in the context of `object_graph`, where a predecessor relation is also provided as argument to `iter`. But maybe here `~from` and `~to` are better instead?

Reply:
`from` and `to` works, or `~src` and `dst`.